src/util/text.ts

```diff
@@ -175,6 +175,35 @@ export function isWhiteSpace(c: i32): bool {
   }
 }
 
+export function isEndOfLiteral(c: i32): bool {
+  switch (c) {
+    case CharCode.COMMA:
```

Comment:
Exclusion lists like these make me a little nervous about forgetting a case. Wondering if there's something more precise we could do?

Reply:
Unfortunate not. It also didn't fix cases like `5=;`, `6,;`, `5 7;`. All this required lookaheads